### DISCUSSION TITLE
Netbox Docker LDAP auth BIN password

### DISCUSSION THREAD

**Discussion by kirillmuravyev** [netbox-community/netbox-docker#1058] [General]
Hello Community!

Recently had some challenges with configuring LDAP auth.
It hasn't been wotking showing below error:
`netbox-docker_netbox_1 WARNING Caught LDAPError while authenticating <username>: INVALID_CREDENTIALS({'msgtype': 97, 'msgid': 1, 'result': 49, 'desc': 'Invalid credentials', 'ctrls': [], 'info': '80090308: LdapErr: DSID-0C090439, comment: AcceptSecurityContext error, data 52e, v4563'})`

BIND acc and pass was fine.

After changing AUTH_LDAP_BIND_PASSWORD to the pass WITHOUT special characters (in particular "$") LDAP auth started to work fine.

Have you had similar issue?

**Comment by soidridine-hub** [comment]
Hello,

This solution helped me a lot. I was facing the same issue with passwords stored in CI/CD variables for pipeline deployment. After replacing the `$` character in the password, everything is now working correctly. Other special characters seem to function without problems.

It appears there might be an issue with how password variables are expanded or with the `netbox.env` file itself.
